internal/cache/util/util.go

```diff
@@ -38,15 +39,16 @@ const (
 	FallbackToGCSErrMsg                       = "read via gcs"
 	FileNotPresentInCacheErrMsg               = "file is not present in cache"
 	CacheHandleNotRequiredForRandomReadErrMsg = "cacheFileForRangeRead is false, read type random read and fileInfo entry is absent"
-	BufferSizeForCRC                          = 65536
 )
 
 const (
-	MiB             = 1024 * 1024
-	KiB             = 1024
-	DefaultFilePerm = os.FileMode(0600)
-	DefaultDirPerm  = os.FileMode(0700)
-	FileCache       = "gcsfuse-file-cache"
+	MiB                        = 1024 * 1024
+	KiB                        = 1024
+	DefaultFilePerm            = os.FileMode(0600)
+	DefaultDirPerm             = os.FileMode(0700)
+	FileCache                  = "gcsfuse-file-cache"
+	BufferSizeForCRC           = 65536
+	MinimumAlignSizeForWriting = 4096
```

Comment:
should we get the MinimumAlignSize using - https://pkg.go.dev/golang.org/x/sys/unix#Getpagesize?

Reply:
I am not sure if there's a link between page size and the align size. If you know could you please point to that ? If not, shall we unblock this PR and continue finding ?

Reply:
Discussed offline, there is no link b/w page size and align size. It's more related to device block-size, as we are keeping minimum to work with all the block-devices.